### PR TITLE
Upgrade to WebAssembly amphtml-validator

### PR DESCRIPTION
**Pull request by antiphoton**
Upgrade `amphtml-validator` to `1.0.35`, which uses the WebAssembly engine.

### CODE REVIEW

File: client/src/test/diagnostics.test.ts [hunk 1]
```diff
@@ -26,83 +26,76 @@ describe('Should get diagnostics for a valid empty ⚡ page', () => {
...
         message:
-          "⚡ : The mandatory tag 'amphtml engine v0.js script' is missing or incorrect. : https://amp.dev/documentation/guides-and-tutorials/learn/spec/amphtml#required-markup",
+          "⚡ : The mandatory tag 'amphtml engine script' is missing or incorrect. : https://amp.dev/documentation/guides-and-tutorials/learn/spec/amphtml/#required-markup",
         range: toRange(17, 7, 17, 12),
         severity: vscode.DiagnosticSeverity.Error,
         source: '⚡',
         code: 'MANDATORY_TAG_MISSING',
       },
-      {
-        message: 'Doctype must be declared first.',
```
Comment (dparikh) [inline client/src/test/diagnostics.test.ts:100]: Why is this removed?

File: client/src/test/diagnostics.test.ts [hunk 2]
```diff
@@ -199,47 +185,23 @@ describe('Should skip non-amp page', () => {
   const docUri = getDocUri('amp/non-amp.html');
 
   it('Skips NON AMP HTML amp/non-amp.html page', async () => {
-    await testDiagnostics(docUri, [
-      {
-        message: 'Doctype must be declared first.',
```
Comment (dparikh) [inline client/src/test/diagnostics.test.ts:204]: Same as earlier comment.
Why all the "Doctype" errors have been removed?

File: client/src/test/diagnostics.test.ts [hunk 3]
```diff
@@ -715,41 +641,27 @@ describe('Should get diagnostics for invalid extensions in ⚡4ads page', () =>
...
       },
-      {
-        message: 'Doctype must be declared first.',
-        range: toRange(0, 0, 0, 4),
-        severity: vscode.DiagnosticSeverity.Warning,
-        source: null,
-        code: 'doctype-first',
-      },
-      {
-        message: '<title> must be present in <head> tag.',
```
Comment (dparikh) [inline client/src/test/diagnostics.test.ts:747]: Why is this removed?

**Review by dparikh — CHANGES_REQUESTED** [review]
Can you also update the https://github.com/diparikh-project/vscode-amphtml-validator/blob/master/README.md with the correct version of amphtml-validator? (1.0.35)

File: client/src/test/diagnostics.test.ts (hunk 1 above)
Comment (antiphoton) [reply client/src/test/diagnostics.test.ts:100]: This error is false positive, because the input file [amp-empty.html](https://github.com/diparikh-project/vscode-amphtml-validator/blob/6628ff875886d54a8b918d82880b54491e932f34/client/testFixture/amp/amp-empty.html) does declare the `Doctype`.
This is removed because the latest validator no longer generate this error.

File: client/src/test/diagnostics.test.ts (hunk 2 above)
Comment (antiphoton) [reply client/src/test/diagnostics.test.ts:204]: Same as earlier, the latest version validator no longer generates this error.

File: client/src/test/diagnostics.test.ts (hunk 3 above)
Comment (antiphoton) [reply client/src/test/diagnostics.test.ts:747]: This error does not exist in the latest validator.

**Comment by antiphoton** [comment]
@dparikh Could please take another look?

**Comment by antiphoton** [comment]
> Can you also update the https://github.com/diparikh-project/vscode-amphtml-validator/blob/master/README.md with the correct version of amphtml-validator? (1.0.35)

Done

**Review by dparikh — APPROVED** [review]
